apps/system/js/ftu_ping.js

```diff
@@ -68,6 +69,9 @@
     // Data used in ping
     _pingData: {},
 
+    // Data used in args
+    _argsData: {},
```

Comment:
suggestion: maybe use `_infoData` instead of `_argsData`, to indicate these are settings that we're pulling for the `info` section of the telemetry payload?